### PR TITLE
Nahiyan_Fixed Header UI

### PR DESCRIPTION
**Pull request by Nahiyan-16**
# Description
Fixed header UI by centering timer and hiding elements based on screen width

## Related PRS (if any):
This frontend PR is related to the dev backend.

## Main changes explained:
- Centered timer
- Font size decrease for timer
- Hiding clock icon when under 530px width
- Hiding owner message when between 1400px and 1700px

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard
6. verify that header UI is working properly from all viewports (300px and up)
7. Make sure there are no overlaps or items going out of view

## Screenshots or videos of changes:
Before: 
Screenshot 2024-04-28 at 12 00 47 PM
Screenshot 2024-04-28 at 12 01 38 PM
Screenshot 2024-04-28 at 12 02 22 PM

### CODE REVIEW

**Review by linkhanthtel — DISMISSED** [review]
New UI looks good to me! Clock icon is hidden under 530px and no components are overlapped! Nice work!

**Review by MinSun0814 — DISMISSED** [review]
New UI looks clean! Text is successfully hidden when resizing the window and clock icon is hidden.
image
image

**Review by hbodgal — DISMISSED** [review]
image

Everything works fine at different viewports.

File: src/components/Timer/Timer.module.css [hunk 1]
```diff
@@ -120,3 +120,20 @@
...
+    grid-template-columns: auto auto;
+  }
+
+  .btns button {
+    padding: 1px 1px;
+  }
+}
+
+@media (max-width: 530px) {
+  .btnDiv{
```
Comment (dkn6) [inline src/components/Timer/Timer.module.css:136]: Thank you for the excellent work! 
Nits: Have you run Prettier?

**Review by dkt6 — CHANGES_REQUESTED** [review]

**Review by dkt6 — COMMENTED** [review]
@Nahiyan-16
I think the update has not been pushed yet.
Could you please check it?

**Comment by Nahiyan-16** [comment]
> @Nahiyan-16 I think the update has not been pushed yet. Could you please check it?

I'm not too sure what changes you are requesting. Can you please clarify?

File: src/components/Timer/Timer.module.css [hunk 2]
```diff
@@ -120,3 +120,20 @@
...
+    display: none;
+  }
+}
```
Comment (dkn6) [inline src/components/Timer/Timer.module.css:139]: > > @Nahiyan-16 I think the update has not been pushed yet. Could you please check it?
> 
> I'm not too sure what changes you are requesting. Can you please clarify?

Sure! If Prettier has been successfully executed, the code would look like this.
```suggestion
@media (max-width: 530px) {
  .btnDiv {
    display: none;
  }
}
```

**Review by dkt6 — APPROVED** [review]

**Comment by one-community** [comment]
Thank you all, merging!

**Pull request merged**